commons/active/banking/banking_attempt.avsc

```diff
@@ -0,0 +1,30 @@
+{
+  "namespace": "org.radarcns.active.banking",
+  "type": "record",
+  "name": "BankingAttempt",
+  "doc": "A Banking App Attempt to complete a transaction, which can contain a PIN, an Amount and a Confirm/Cancel Transaction event. It can also time out or end.",
```

Comment:
Why not call this just transaction? And then the transaction can still fail or be exited. In the doc, you could describe that one transaction may be described by multiple events.

Reply:
Perhaps rephrase to:
`Transaction in Banking App. This captures a full transaction: inputting a PIN number, then an amount and then a confirmation or cancellation. A transaction may also time out or end prematurely.`

Reply:
This is also corrected.